src/main/java/org/prebid/server/handler/CookieSyncHandler.java

```diff
@@ -52,24 +54,29 @@
     private final long defaultTimeout;
     private final UidsCookieService uidsCookieService;
     private final BidderCatalog bidderCatalog;
+    private final List<List<String>> listOfPrioritisedBidders;
     private final GdprService gdprService;
     private final Integer gdprHostVendorId;
     private final boolean useGeoLocation;
+    private final boolean defaultCoop;
     private final AnalyticsReporter analyticsReporter;
     private final Metrics metrics;
     private final TimeoutFactory timeoutFactory;
 
     public CookieSyncHandler(String externalUrl, long defaultTimeout, UidsCookieService uidsCookieService,
-                             BidderCatalog bidderCatalog,
+                             BidderCatalog bidderCatalog, List<List<String>> listOfPrioritisedBidders,
                              GdprService gdprService, Integer gdprHostVendorId, boolean useGeoLocation,
-                             AnalyticsReporter analyticsReporter, Metrics metrics, TimeoutFactory timeoutFactory) {
+                             boolean defaultCoop, AnalyticsReporter analyticsReporter, Metrics metrics,
+                             TimeoutFactory timeoutFactory) {
         this.externalUrl = HttpUtil.validateUrl(Objects.requireNonNull(externalUrl));
         this.defaultTimeout = defaultTimeout;
         this.uidsCookieService = Objects.requireNonNull(uidsCookieService);
         this.bidderCatalog = Objects.requireNonNull(bidderCatalog);
+        this.listOfPrioritisedBidders = listOfPrioritisedBidders;
```

Comment:
i think it is good to check the value for null since this is configuration related.